task/minio-cli/0.1/README.md

```diff
@@ -0,0 +1,67 @@
+## MinIO CLI Task
+
+[MinIO](http://minio.io/) is a High Performance Object Storage released under Apache License v2.0.
+It is API compatible with Amazon S3 cloud storage service.
+
```

Comment:
Can you please also add the doc which points to the set of commands possible with the `minio-cli`. https://docs.min.io/docs/minio-client-complete-guide I guess this is the one

Reply:
done!